src/metabase/channel/params.clj

```diff
@@ -13,15 +13,18 @@
   "Substitute parameters in text with values from context.
 
   Params are specified using mustache syntax, e.g. {{param}}."
-  [text context]
+ ([text context]
+  (substitute-params text context false))
+ ([text context ignore-missing?]
```

Comment:
I'm not sure if there's any practical reason we might add more options in future, but have a naked flag argument is typically a smell and I would prefer kwargs. `(foo! false)` is so much less obvious that `(foo! :fire-missiles false)`.

Reply:
yep, I do hate naked flag args :D. updated.